v2/tools/asoctl/internal/importing/importable_arm_resource.go

```diff
@@ -220,13 +293,39 @@ func (i *importableARMResource) importChildResources(
 		containerURI,
 		imp.GetAPIVersion())
 	if err != nil {
+		var responseError *azcore.ResponseError
+		if errors.As(err, &responseError) {
```

Comment:
_Almost_ the same error processing logic as above? Would these two places likely evolve in tandem? Can we share the logic of error classification such that each spot can use the one function (but possibly omit its own logs, or something?)

Or maybe we really want a log on the forbidden case above too?